packages/vc-verification/src/revocation-verification.ts

```diff
@@ -68,14 +60,21 @@ export class RevocationVerification {
       }
     } else if (revokerRole) {
       try {
-        const revokerVC = await this.issuerVerification.verifyIssuance(
+        const revokerCredential = await this.issuerVerification.verifyIssuance(
           revoker,
           revokerRole
         );
-        await this.issuerVerification.verifyIssuer(
-          issuerDID(revokerVC.issuer),
-          revokerRole
-        );
+        if (revokerCredential.issuer) {
```

Comment:
When `RevocationVerification` gets issuer credential it could be OffchainClaim or VerifiableCredential. Ideally `RevocationVerification` should use IssuerVerification interface and not be aware of implementation of this interface and thus avoid this `if` checks